app-misc/trash-cli/trash-cli-0.23.9.23.ebuild

```diff
@@ -3,25 +3,31 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9,10} )
-
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..12} )
```

Comment:
I would consider dropping at least python 3.9 here, maybe even 3.10. 3.11 is stable on amd64 and x86, this package seems to be a standalone tool and is going into testing.

Reply:
Since 3.9 and up are all available in Gentoo I think I'll keep it for now. I'll drop it when older Python versions are dropped from the portage repository.